index.js

```diff
@@ -14,13 +14,18 @@ const {
     const region = core.getInput('region')
     const groupName = core.getInput('group_name')
     const newTaskDefinitionArn = core.getInput('new_task_definition_arn')
+    const exceptions = core.getMultilineInput('exceptions') || [];
     const scheduler = new SchedulerClient({ region });
 
     console.log(`Getting schedules in the group ${groupName}.`);
-    const { Schedules: schedules } = await scheduler.send(
+    let { Schedules: schedules } = await scheduler.send(
       new ListSchedulesCommand({ GroupName: groupName })
     );
 
+    schedules = schedules.filter(
+      (schedule) => !exceptions.includes(schedule.Name)
+    );
```

Comment:
@zachfey This is where we are filtering out any exceptions passed to not update.